### PR TITLE
handle all exception in ssh connection test

### PR DESCRIPTION
**Pull request by keedya**

### CODE REVIEW

**Comment by keedya** [comment]
@anhou @yyscamper @brianparry @tannoa2 @geoff-reid @uppalk1

**Comment by keedya** [comment]
@yyscamper, @anhou  I run the test on my personal jenkins server:
you can find it here:
http://10.240.19.232:8080/job/Continuous-Test/job/Continuous-Functional/14185/testReport/(root)/

File: test/tests/api/v1_1/decomission_node_tests.py [hunk 1]
```diff
@@ -107,11 +107,13 @@ def worker(ssh_conn, delay, retries):
...
-                except pxssh.ExceptionPxssh  as error:
-                    LOG.info('ssh login error {0}'.format(error))
-                    #log error
+                except Exception  as error:
+                    LOG.info('ssh login error for node {0}, hostname {1}'
+                            .format(ssh_conn.node_id, ssh_conn.host))
                     time.sleep(delay)
-            LOG.warning('could not connet to host {0} node id {1}'.format(ssh_conn.hostname, ssh_conn.node_id))
+                    pass
+            LOG.warning('could not connet to host {0} node id {1}'
```
Comment (anhou) [inline test/tests/api/v1_1/decomission_node_tests.py]: connect

File: test/tests/api/v1_1/decomission_node_tests.py [hunk 2]
```diff
@@ -107,11 +107,13 @@ def worker(ssh_conn, delay, retries):
                     ssh_conn.is_up = True
                     LOG.info('set it to True {0}'.format(ssh_conn.is_up))
                     return True 
-                except pxssh.ExceptionPxssh  as error:
-                    LOG.info('ssh login error {0}'.format(error))
-                    #log error
+                except Exception  as error:
+                    LOG.info('ssh login error for node {0}, hostname {1}'
```
Comment (anhou) [inline test/tests/api/v1_1/decomission_node_tests.py]: LOG.error instead of LOG.info？

**Comment by anhou** [comment]
Thanks @keedya posting the test log, although I cannot access it  :),  👍  with some minor comments

File: test/tests/api/v1_1/decomission_node_tests.py (hunk 2 above)
Comment (keedya) [inline test/tests/api/v1_1/decomission_node_tests.py]: it is not an error for this test

Comment (anhou) [inline test/tests/api/v1_1/decomission_node_tests.py]: Gotta, thanks!

Comment (keedya) [inline test/tests/api/v1_1/decomission_node_tests.py]: http://10.240.19.232:8080/job/Continuous-Test/job/Jenkins-Sandbox/1/testReport/(root)/

**Comment by keedya** [comment]
@anhou, I addressed your comments thanks!!

**Comment by anhou** [comment]
@keedya it's so late in your timezone, so hardworking!! thanks for your quick feedback! I'll merge immediately after Jenkins finiish

**Pull request merged**